app/models/course/assessment/category.rb

```diff
@@ -40,7 +42,7 @@ def build_initial_tab
 
   def build_initial_folder
     build_folder(name: title, course: course, parent: course.root_folder,
-                 start_at: Time.zone.now)
+                 start_at: Time.zone.now) unless folder
```

Comment:
@lowjoel this is still in `before_validation` is because folder need attributes like title, start_at, etc.